Classes/item.rb

```diff
@@ -0,0 +1,23 @@
+require 'date'
+
+class Item
+  attr_accessor :archived, :label, :author, :id, :publish_date, :genre
+
+  def initialize(publish_date, archived: false, id: nil)
+    @genre = ''
+    @author = ''
+    @source = ''
+    @label = ''
```

Comment:
- [ ] It is required that properties for the 1-to-many relationships (genre, author, label) should NOT be set in the constructor method. Instead, they should have a custom setter method created. hint:

```ruby
def genre=(genre)
      @genre = genre
      genre.items.push(self) unless genre.items.include?(self)
 end
```

Reply:
done, thanks!